### ISSUE TITLE
Global Color Palette Change

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1352]
**Component block:** none, global change

**Design:** https://www.figma.com/design/T2bgro0OiuLqxvm1uLhJZ2/Clark-County-NV?node-id=93-224&p=f&t=7ZXrRUQI9M74s5dW-0

As a site visitor, I want the County colors on the website to always be accessible and meet color constrast requirements so that I can easily view and find anything I need no matter my disability or impairment

**Requirements:** 
Update the global color palette variables to be used by all component blocks
Primary color ramp should have the following hex codes: 
Primary orange 900: #722922
Primary orange 600: #963821
Primary orange 300: #C54B2A

Primary blue 900: #0093B2
Primary blue 600: #76C5D6
Primary blue 300: #BFDCE0

Secondary blue 800: #243772
Secondary blue 600: #365EA5

Secondary orange 800: #B86125
Secondary orange 600: #FFB434

Neutral ramp black: #232222
Neutral ramp 800: #545454
Neutral ramp 600: #828282
Neutral ramp 400: #C4C4C4
Neutral ramp 200: #EEEEEE
Neutral ramp white: #FFFFFF

**Acceptance Criteria:** 
1. Given I am viewing a web page, when the page renders, then elements on the page adhere to the three defined primary colors. 
2. Given I am viewing a web page, when the page renders, then elements on the page that display in the blue colors adhere to the three defined primary blue colors. 
3. Given I am viewing a web page, when the page renders, then elements on the page that display in the secondary blue colors adhere to the three defined secondary blue colors. 
4. Given I am viewing a web page, when the page renders, then elements on the page adhere to the three defined secondary colors. 
5. Given I am viewing a web page, when the page renders, then elements on the page adhere to the defined neutral colors.

**Comment by beccasuchanek** [comment]
@aaronstewart-dev This ticket passed all ACs, see details below:

**Test Page URL:** https://develop--clarkcountynv--aemsites.aem.page/drafts/bsuchanek/kitchen-sink (used the inspect window to see global color styles)

**Status of ACs**
1. Given I am viewing a web page, when the page renders, then elements on the page adhere to the three defined primary colors.

<img width="320" height="79" alt="Image" src="https://github.com/user-attachments/assets/1883a563-6b3b-46be-8b46-98b819cdd3d8" />

2. Given I am viewing a web page, when the page renders, then elements on the page that display in the blue colors adhere to the three defined primary blue colors.

<img width="284" height="54" alt="Image" src="https://github.com/user-attachments/assets/7fa263cc-094e-4ecd-bf99-09fdd6b3a378" />

3. Given I am viewing a web page, when the page renders, then elements on the page that display in the secondary blue colors adhere to the three defined secondary blue colors.

<img width="270" height="35" alt="Image" src="https://github.com/user-attachments/assets/d08b8c5c-60d8-4e90-a071-2e7d028bf968" />

4. Given I am viewing a web page, when the page renders, then elements on the page adhere to the three defined secondary colors.

<img width="305" height="51" alt="Image" src="https://github.com/user-attachments/assets/2db380f4-366e-438f-a250-c80dcfd6cc24" />

5. Given I am viewing a web page, when the page renders, then elements on the page adhere to the defined neutral colors.

<img width="272" height="104" alt="Image" src="https://github.com/user-attachments/assets/ff3b5e49-93bf-447e-bf1a-4f3b4032899d" />
